docs/UserGuide.md

```diff
@@ -196,32 +198,55 @@ Edits an existing application in SoC InternApply.
-
+- To edit the details of an application, you can follow this format (adding \n to type in a new line): `edit 1 d/Example details \nThis is a newline of the details`<br>e.g.`edit 1 d/This company requires a preliminary coding round.\n I should practice more on HackerRank` will result in this details being added:
 ```
 This company requires a preliminary coding round. 
 I should practice more on HackerRank
 ```
+#### Advanced Function
+
+Removing Tags from existing application.
+
+**Format**: `edit INDEX [t/removetags] [t/removepriority] [t/removestatus]`
+
+**CAUTION**: While it is possible to use these special Tags well editing an existing application in SoC InternApply, we highly recommend not doing so unless you know what you are doing. However, if you wish to do so please refer to the description below beforehand.
+
+- Removes the Tags from an existing application.
+- `[t/removetags]` will remove all Optional Tags from an application (i.e. Tags with the prefix `t/`).
+- `[t/removepriority]` will remove the Priority Tags from an application (i.e. Tags with the prefix `pt/`).
+- `[t/removestatus]` will remove the Application Status Tags from an application (i.e. Tags with the prefix `ast/`).
+- These special Tags are case-insensitive (i.e. typing `t/ReMOveTagS` would also work). 
+- These special Tags cannot be used as a valid Optional Tag for your applications.
+- These special Tags can be used in any order, and you can either use 1 of them, a pair of them, or all 3 of them.
+- These special Tags takes precedence over all other Tag inputs. Refer to the Example usage below for reference.
+- You can still use these special Tags even if the respective Tags do not exist for the specified application. Nothing will happen in this case.
+- If removing any of the Tags from an application causes it to become a duplicate application the command will be invalid. Please refer to our Notes about duplicate applications
+
+**Example usages and expected outcomes**
+
+- `edit 1 t/removetags t/removepriority` Removes all the Optional Tags and the Priority Tag from the 1st application.
+- `edit 1 p/65258719 t/removestatus` Removes the Application Status Tag from the 1st application. Edits the phone number of the 1st application to be `65258719`
+- `edit 1 pt/HIGH t/removepriority` Special Tags takes precedence over all other Tag inputs. Removes the Priority Tag from the 1st application. Ignores the `pt/HIGH` input.
+- `edit 1 t/removestatus t/Singapore t/ExEmployee` Special Tags takes precedence over all other Tag inputs. Removes the Application Status Tag from the 1st application. Ignores the `t/Singapore t/ExEmployee` inputs.
+- `edit 1 t/removestatus` followed by `edit 1 t/removestatus` First removes the Application Status Tag from the 1st application. Then does nothing since there does not exist an Application Status Tag for the 1st application. 
```

Comment:
LGTM, smart separating more complicated functions into a different section, might want to consider separating edit function here too